src/dlangui/graphics/resources.d

```diff
@@ -205,6 +205,7 @@ EmbeddedResource[] embedResource(string resourceName)() {
 }
 
 /// embed all resources from list
+pragma(inline, false) // needed because https://issues.dlang.org/show_bug.cgi?id=24052
```

Comment:
This should be for DMD only, as neither LDC not GDC have this issue. Please change to
```d
version(DigitalMars)
{
    pragma(inline, false);
}
```

Reply:
Done